contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-19
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Explainers
+---
+
+Every startup loves to see signups, subscriptions, and revenue go up, yet all engineers basically work on things indirectly related to doing so. The power of growth engineers is that they focus on making these numbers go up, working on whatever it takes.
+
+A growth engineer is an engineer who works on projects impacting the core business metrics, no matter where they are in the product. This is different than regular engineers who work on specific products and features to improve metrics for them. This indirectly improves the core business metrics.
```

Comment:
Would be good to disambiguate what "core business metrics" means. e.g. acquisition metrics, activation metrics, revenue?

Reply:
maybe "growth metrics" is a better term and defining what that is?

Reply:
went with growth metrics for now.